### PR TITLE
fix #60944 cepmuzikindir.pro

### LINKED ISSUE #60944: cepmuzikindir.pro
### Issue URL (Ads)
 [https://cepmuzikindir.pro/arama.php?q=refleks+sen+hep+benimsin](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fcepmuzikindir.pro%2Farama.php%3Fq%3Drefleks%2Bsen%2Bhep%2Bbenimsin)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.4.31
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (1440), <br/>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> NoCoin Filter List<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by stanislav-atr**
#60944

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -116,6 +116,8 @@ aykutozdemir.com.tr##.theiaStickySidebar > #text-5
 aykutozdemir.com.tr##.theiaStickySidebar > #text-10
 7deniz.net###lock > div[style] > a[href][target="_blank"] > img
 ||7deniz.net/resimler/7_deniz_*_kapak_tek-banner_*.gif
+||kingads.mobi/js/nemp3-tek.js$domain=cepmuzikindir.pro
```
Comment (zloyden) [inline TurkishFilter/sections/specific.txt:119]: Remove a domain modificator.

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -116,6 +116,8 @@ aykutozdemir.com.tr##.theiaStickySidebar > #text-5
...
+cepmuzikindir.pro##a[href*="kingads.mobi"]
```
Comment (zloyden) [inline TurkishFilter/sections/specific.txt:120]: It's kinda wide and a bit out of our syntax in filters. Don't skip useful parts.
`cepmuzikindir.pro##a[href^="https://kingads.mobi/"]`

**Review by zloyden — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (AdamWr) [reply TurkishFilter/sections/specific.txt:119]: It seems that similar scripts are also used on another websites - https://publicwww.com/websites/%22kingads.mobi%22/, so I think that we could add it to `adservers`.
`||kingads.mobi^$third-party`

**Review by AdamWr — APPROVED** [review]

File: TurkishFilter/sections/specific.txt (hunk 1 above)
Comment (zloyden) [reply TurkishFilter/sections/specific.txt]: Good catch, I agree:)

**Review by zloyden — APPROVED** [review]

**Pull request merged**